### PR TITLE
Add aap_remove role

### LINKED ISSUE #66: Add roles to remove controller and ah
Transform the attached playbooks into roles, possibly with one or two wrapper playbooks to use them directly from the collection:

[ah_remove.yml.txt](https://github.com/redhat-cop/tower_utilities/files/8222427/ah_remove.yml.txt)
[controller_remove.yml.txt](https://github.com/redhat-cop/tower_utilities/files/8222428/controller_remove.yml.txt)

### PR DESCRIPTION
**Pull request by Tompage1994**
### What does this PR do?
Adds role to remove AAP controller and AH

### How should this be tested?
Run the sample playbook given in README

### Is there a relevant Issue open for this?
resolves #66

### CODE REVIEW

File: roles/aap_remove/tasks/ah_remove.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - nginx
+  become: true
+
+- name: Remove added groups
+  group:
+    name: "{{ item }}"
+    state: absent
+  loop:
+    - redis
+  become: true
```
Comment (djdanielsson) [inline roles/aap_remove/tasks/ah_remove.yml:67]: missing ... to end the yaml file

File: roles/aap_remove/tasks/controller_remove.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- name: Remove added groups
+  group:
+    name: "{{ item }}"
+    state: absent
+  loop:
+    - redis
+    - nginx
+    - cgred
+  become: true
```
Comment (djdanielsson) [inline roles/aap_remove/tasks/controller_remove.yml:67]: missing ... to end the yaml file

**Review by djdanielsson — CHANGES_REQUESTED** [review]
if you could just fix the 2 files missing the ending notation the rest looks good

**Comment by Tompage1994** [comment]
> if you could just fix the 2 files missing the ending notation the rest looks good

Done

**Review by djdanielsson — APPROVED** [review]
LGTM

**Pull request merged**